src/main/DomainLayer/TradeControl.py

```diff
@@ -16,7 +16,7 @@ class TradeControl:
     __instance = None
 
     @staticmethod
-    def getInstance():
```

Comment:
Let's refactor it later? I think you were missing some updates when you were refactoring this.